pyenvisalink/dsc_client.py

```diff
@@ -30,9 +30,8 @@ def dump_zone_timers(self):
         self.send_command(evl_Commands['DumpZoneTimers'], '')
 
     def keypresses_to_partition(self, partitionNumber, keypresses):
-        """Send keypresses to a particular partition."""
-        for char in keypresses:
-            self.send_command(evl_Commands['PartitionKeypress'], str.format("{0}{1}", partitionNumber, char))
+        """Send keypresses (max of 6) to a particular partition."""
+	self.send_command(evl_Commands['PartitionKeypress'], str.format("{0}{1}", partitionNumber, keypresses[:6]))
```

Comment:
I know this is incredibly nit-picky, but the indentation of the "self.send_command". doesn't match the comment above. Can you space it in 1 more?